include/boost/geometry/core/config.hpp

```diff
@@ -0,0 +1,20 @@
+// Boost.Geometry
+
+// Copyright (c) 2018 Oracle and/or its affiliates.
+// Contributed and/or modified by Adam Wulkiewicz, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_CORE_CONFIG_HPP
+#define BOOST_GEOMETRY_CORE_CONFIG_HPP
+
```

Comment:
Cool, I've added this file too - for removing rescaling.
Your's will be merged earlier so I will adapt/extend it